gnome-extra/gnome-commander/gnome-commander-1.10.0.ebuild

```diff
@@ -52,6 +51,13 @@ src_configure() {
 		$(use_with unique)
 }
 
+pkg_pretend() {
+	if tc-is-gcc && [[ $(gcc-major-version) -lt 8 ]]; then
```

Comment:
You aren't inheriting the needed eclass.

Reply:
Ah, right. Fixed it. I am wondering why this wasn't detected by the automatic checks and also not when I locally tested it. Hm...